src/ducks/SANCharts/ChartPage.js

```diff
@@ -29,24 +29,41 @@ const DEFAULT_TIME_RANGE = '6m'
 
 const { from, to } = getIntervalByTimeRange(DEFAULT_TIME_RANGE)
 
+const DEFAULT_STATE = {
+  timeRange: DEFAULT_TIME_RANGE,
+  from: from.toISOString(),
+  to: to.toISOString(),
+  slug: 'santiment',
+  metrics: ['historyPrice'],
+  title: 'Santiment (SAN)',
+  interval: '1w'
+}
+
 class ChartPage extends Component {
-  mapQSToState = ({ location: { search } }) => {
-    const data = qs.parse(search, { arrayFormat: 'comma' })
-    if (typeof data.metrics === 'string') {
-      data.metrics = [data.metrics]
+  constructor (props) {
+    super(props)
+
+    let passedState
```

Comment:
Possible better to move it as separate func, with props as the input.
Actually this func will be easy to test.
@DmitriVanGuard